.github/CODEOWNERS

```diff
@@ -105,3 +105,18 @@ sycl/test-e2e/KernelFusion @victor-eds @Naghasan @sommerlukas
 
 # Matrix tests
 sycl/test-e2e/Matrix @dkhaldi @YuriPlyakhin @yubingex007-a11y
+
+#Native CPU PI 
+sycl/plugins/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers
```

Comment:
@stdale-intel, could you make sure that the team has write access to the repository, please?